ArcGISRuntimeSDKQt_CppSamples/Analysis/ViewshedLocation/ViewshedLocation.qml

```diff
@@ -0,0 +1,304 @@
+                                verticalCenter: parent.verticalCenter
+                            }
+                            checked: true
+                            onCheckedChanged: viewshedSample.viewshedVisible = checked;
+                        }
+                    }
+
+                    Item {
+                        width: parent.width
+                        height: 25 * scaleFactor
+
+                        Text {
+                            anchors.verticalCenter: parent.verticalCenter
+                            width: parent.width * 0.75
+                            text: qsTr("Frustum Visible")
+                            font.pixelSize: 14 * scaleFactor
+                        }
+
+                        Switch {
+                            anchors {
+                                right: parent.right
+                                margins: 10 * scaleFactor
+                                verticalCenter: parent.verticalCenter
+                            }
+                            checked: false
+                            onCheckedChanged: viewshedSample.frustumVisible = checked;
+                        }
+                    }
+
+                    ViewshedSlider {
+                        titleText: qsTr("Min Distance (m)")
+                        parameterValue: viewshedSample.minDistance
+                        minValue: 1
+                        maxValue: 2000
+                        onParameterValueChanged: viewshedSample.minDistance = parameterValue;
+                    }
+
+                    ViewshedSlider {
+                        titleText: qsTr("Max Distance (m)")
+                        parameterValue: viewshedSample.maxDistance
+                        minValue: 1
+                        maxValue: 2000
+                        onParameterValueChanged: viewshedSample.maxDistance = parameterValue;
+                    }
+
+                    ViewshedSlider {
+                        titleText: qsTr("Vertical Angle")
+                        parameterValue: viewshedSample.verticalAngle
+                        minValue: 1
+                        maxValue: 179
+                        onParameterValueChanged: viewshedSample.verticalAngle = parameterValue;
+                    }
+
+                    ViewshedSlider {
+                        titleText: qsTr("Horizontal Angle")
+                        parameterValue: viewshedSample.horizontalAngle
+                        minValue: 1
+                        maxValue: 179
+                        onParameterValueChanged: viewshedSample.horizontalAngle = parameterValue;
+                    }
+
+                    ViewshedSlider {
+                        titleText: qsTr("Heading")
+                        parameterValue: viewshedSample.heading
+                        minValue: 1
+                        maxValue: 359
+                        onParameterValueChanged: viewshedSample.heading = parameterValue;
```

Comment:
Heading would be nice as a 360 dial, but not sure if one exists in Qt Quick Controls